README.md

```diff
@@ -30,6 +31,7 @@ BCSS [7] | Breast | H&E | [data](https://bcsegmentation.grand-challenge.org/), [
 Bone-Marrow-Cytomorphology [68] | Marrow | May-Grünwald-Giemsa/Pappenheim | [data](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=101941770#101941770bcab02c187174a288dbcbf95d26179e8), [paper](https://ashpublications.org/blood/article/138/20/1917/477932/Highly-accurate-differentiation-of-bone-marrow) | 171.375 cells from 945 patients | images + label | classi (21) | patch (250x250 - single cell) | 40x | 2021 
 BRACS [62] | Breast | H&E | [data](https://www.bracs.icar.cnr.it/details/), [paper](https://arxiv.org/pdf/2111.04740.pdf) | 547 wsi, 4539 ROIs, 189 Patients | images + label (6 subtypes tumor + normal) | classi (7) | wsi + patch | 40x - Aperio AT2 | 2021
 BreakHis [8] | Breast |	H&E	| [data](https://web.inf.ufpr.br/vri/databases/breast-cancer-histopathological-database-breakhis/), [paper](https://www.inf.ufpr.br/lesoliveira/download/TBME-00608-2015-R2-preprint.pdf) |	7.909 (2480 benign, 5429 malignant)	| images + binary label + tumor type (8) (multiple magnifications: 40x, 100x, 200x, 400x) |	classi |	Patch (700x460) | 40x, 100x, 200x, 400x | 2016
+BRCA-M2C [95] | breast | H&E | [data](https://drive.usercontent.google.com/download?id=1HvIzsOs5FP9OdlJKAnU_PM6tX4B2Q1rk&export=download&authuser=0), [paper](https://arxiv.org/abs/2110.04886) |  120 patches belonging to 113 patients (from TCGA) | images+point annotation |  multi-class cell detection | patch (500x500) | 20x | 2021
```

Comment:
Add [github](https://github.com/TopoXLab/Dataset-BRCA-M2C)
Maybe specify: train: 80, valid: 10, test: 30 patches
Maybe add `around 500x500` as apparently not all images are of the same size.